client/package.json

```diff
@@ -71,6 +75,7 @@
 		"eslint-plugin-prettier": "^4.2.1",
 		"eslint-plugin-react": "^7.30.1",
 		"eslint-plugin-react-hooks": "^4.6.0",
+		"gh-pages": "^4.0.0",
```

Comment:
Is this setup for the future? I'm not seeing any other changes in this PR related to this.

Reply:
gh-pages setup is for demo page issue. It will be used in the future.